nilearn/datasets/utils.py

```diff
@@ -774,6 +765,49 @@ def _fetch_files(data_dir, files, resume=True, mock=False, verbose=1):
     return files_
 
 
+def _download_or_mock_file(abort, dl_file, file_, mock, target_file,
+                           temp_target_file, url):
+    if mock:
```

Comment:
Could you add a one-line function description to explain in a nutshell what the function does ?